### PR TITLE
Elias/readme update and small fix

### PR DESCRIPTION
**Pull request by eliasdefaria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,126 @@
...
+    aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+    aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+```
+<!-- end usage -->
+
+# Scenarios
+
+[Run Compile on PR](#run-compile-on-pr-creation-and-update)
+[Run Checks on PR with Snowflake Provider](#run-compile-on-pr-creation-and-update-with-a-snowflake-provider)
+[Push to SDF Cloud on Merge to Main](#push-to-sdf-cloud-on-merge-to-main-with-a-redshift-provider)
```
Comment (xuliangs) [inline README.md:67]: nit: these 3 are on 1 line, is this intended?

**Review by xuliangs — APPROVED** [review]

**Comment by xuliangs** [comment]
Can you label your PR as `documentation` before merged?

File: action.yml [hunk 2]
```diff
@@ -24,10 +24,6 @@ inputs:
     description: "Whether the workspace is dbt based or not"
     required: false
     default: ""
-  dbt_target:
-    description: "The dbt target"
-    required: false
-    default: "dev"
```
Comment (xuliangs) [inline action.yml:30]: Just seeing tests are broken, probably related to this

File: README.md (hunk 1 above)
Comment (eliasdefaria) [reply README.md]: Just updated to make it more clear. The VSCOde table of contents auto gen was taking over when I tried to make them a list. Very annoying

**Pull request merged**